Readme.md

```diff
@@ -97,9 +97,10 @@ at_exit do
     undo_something
   end
 end
-
 ```
 
+_note:_ when using the standalone runners (`parallel_rspec`, `parallel_test`, ...) then the `ParallelTests` module isn't available.
```

Comment:
I though the fix would be to drop a `require 'parallel_tests'` at the top of the example ?

Reply:
And I thought you said it might not even be available as it's not even necessarily in the Gemfile. Although, then it should still have access to the gem and if it ran with `bundle exec` the command itself shouldn't work so it might work... :thinking: 

Eh, I'm unsure but you're probably right. Good we talked about it. Say the word and I'll change it again :)

Reply:
Added the explicit require